deploy/examples/csi/rbd/storageclass.yaml

```diff
@@ -18,6 +18,9 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: rook-ceph-block
+  # annotation:
+  #   storageclass.kubernetes.io/is-default-class: true
+    storageclass.kubernetes.io/is-default-class: true
```

Comment:
Please [squash the commits](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-history) and also remove this line.

Reply:
Dear @travisn,

Thanks for your helpful comment. 

Sincerely,
Mostafa